React/Modules/RCTStatusBarManager.m

```diff
@@ -103,6 +103,26 @@ - (void)applicationWillChangeStatusBarFrame:(NSNotification *)notification
   }]);
 }
 
+RCT_EXPORT_METHOD(getStatusBarOrientation:(RCTPromiseResolveBlock)resolve
+                                   reject:(RCTPromiseRejectBlock)reject)
+{
+  NSDictionary *orientations = @{
```

Comment:
Hello, I'm having troubles to invoque RCTConvert in the returned value, can you help me, please?

The setup:
```
RCT_ENUM_CONVERTER(UIInterfaceOrientation, (@{
  @"portrait": @(UIInterfaceOrientationPortrait),
  @"portrait-upside-down": @(UIInterfaceOrientationPortraitUpsideDown),
  @"landscape-left": @(UIInterfaceOrientationLandscapeLeft),
  @"landscape-Right": @(UIInterfaceOrientationLandscapeRight),
}), UIInterfaceOrientationUnknown, integerValue);
```

```
RCT_EXPORT_METHOD(getInterfaceOrientation:(RCTResponseSenderBlock)callback)
{
  callback(@[@{
    @"orientation": @([[UIApplication sharedApplication] statusBarOrientation]),
  }]);
}
```

I've seen many examples inside the source code of other components where converting input args, but none for converting returned values, how can I invoque the converter? For example:

Convert `UIInterfaceOrientationPortrait` --> `"portrait"`

Thanks